scala/src/main/scala/com/quby/demo/databricks/storage/Repository.scala

```diff
@@ -3,11 +3,59 @@ package com.quby.demo.databricks.storage
 import com.quby.demo.databricks.schema.{ElectricityPowerSchema, RawSchema}
 import org.apache.spark.sql.Dataset
 
+/***
+  * Repository is the interface we use to access data from the storage.
+  *
+  * The two implementations are:
+  *  * SampleRepository -> Use this one for generating sample data on-the-fly
+  *  * DeltaRepository -> Use this one for accessing data stored on Databricks Delta
+  */
 trait Repository {
-  def raw(dateFrom: String, dateTo: String): Dataset[RawSchema]
 
-  def electricityPower(dateFrom: String, dateTo: String): Dataset[ElectricityPowerSchema]
+  /***
```

Comment:
according to the scala style guid you make a docstring with 2 *'s. 
https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html

Reply:
Nice catch :)